Local/Puppet3.py

```diff
@@ -24,7 +24,7 @@ def handle(self, message):
         if days > 0:
             txt = "Puppet 3 is End of Life in %s Days" % days
```

Comment:
It will, but that interpolation syntax is just a minor style thing. Not even sure why it got deprecated in favor of `format()`